modules/programs/nushell.nix

```diff
@@ -141,10 +151,14 @@ in {
         ];
       })
 
-      (mkIf (cfg.envFile != null || cfg.extraEnv != "") {
+      (let
+        envVarsStr = concatStringsSep "\n"
+          (mapAttrsToList (k: v: "let-env ${k} = ${v}") cfg.envVars);
```

Comment:
This is a bit confusing. You'd presume you can write something like:

```
EDITOR = "${pkgs.vim}/bin/vim"
```
But you'll have to do something like:

```
EDITOR = ''
  "${pkgs.vim}/bin/vim"
''
```
Or use escaping:

```
EDITOR = "\"${pkgs.vim}/bin/vim\""
```

It would be nice if the option would do this by default.

This is in line with an earlier PR that provides arbitrary settings. By default the literal value is used, but you can use .__nu to write Nu code instead. See https://github.com/nix-community/home-manager/pull/3831

Reply:
I've just tested to make sure with `    EDITOR = "${pkgs.helix}/bin/hx";`
and it works like a charm.

Reply:
Oh right, without spaces it is fine of course. Good point 👍